src/test/java/tests/TurboMilestoneTest.java

```diff
@@ -34,6 +32,30 @@ public void turboMilestoneTest() {
     }
 
     @Test
+    public void getDueDateComparator_milestoneOrdering() {
```

Comment:
Can we have several open milestones with due date and several closed milestones with due date here? Currently, the status of the milestones are not set. Should be e.g. 2 open milestones w/ due date, 2 closed milestones w/ due date (you might want these 4 to overlap), 2 open w/o due date and 2 closed w/o due date.